client/src/pages/app/index.tsx

```diff
@@ -256,7 +261,10 @@ export const App = () => {
                       onChange={
                         selectedDb.value.name !== 'Create new database'
                           ? setSelectedDb
-                          : history.push('/create-database')
+                          : history.push({
+                              pathname: '/create-database',
+                              state: app.name,
```

Comment:
just wondering, shouldn't state be an object?

Reply:
it can be, but works w/o being object as well.